src/containers/VisitPage.js

```diff
@@ -72,24 +72,29 @@ class VisitPage extends Component {
 
     this.steps = [
       {
-        title: 'Current Benefits',
-        form:  CurrentBenefitsStep,
+        displayName: 'Current Benefits',
```

Comment:
Not blocking, but this may be better as `tabName` if we're going to change the prop name. Is this related to the snippets changes?

Reply:
Forget this one, look at the next one instead. I just saw your note about what you were trying to do with this. I'm still not sure it's part of the snippet changes, but meh.

Reply:
We can keep it as `title` actually...I changed it to try to make it more clear that this was a display string as opposed to an internal identifier, but that's probably not necessary.

Although we could actually probably get rid of this field entirely, and instead add the title to the snippets for each step and use `snippets[ step.key ].title` for the tab text.

Reply:
Or `tabTitle` to keep it unambiguous

Reply:
Actually I guess it would be better to have all the tab names together in one section of the snippets